packages/components/src/components/Table/Table.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import { Meta, StoryObj } from '@storybook/react';
+
+import { Table as TableComponent, allowedTableFrameProps } from './Table';
+import { getFramePropsStory } from '../../utils/frameProps';
+
+const EXAMPLE_TOKENS = [
+    { name: 'USDT', balance: '10', price: '$1,00' },
+    { name: 'USDC', balance: '100', price: '$1,00' },
+    { name: 'MANA', balance: '20', price: '$2,15' },
+];
+
+const meta: Meta = {
+    title: 'Table',
+} as Meta;
+export default meta;
+
+export const Table: StoryObj = {
+    render: props => (
+        <TableComponent {...props}>
```

Comment:
nit: you can use

```tsx
const { Cel,HeaderCell,HeaderRow } = TableComponent

Reply:
If you don't mind I like to use the . Cell etc. I like it more 😁